### PR TITLE
plan changes blog

### PR DESCRIPTION
**Pull request by hugovalente-pm**

### CODE REVIEW

**Review by Aliki92 — APPROVED** [review]

File: blog/2023-11-02-netdata-plan-changes/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+![plans](https://github.com/netdata/blog/assets/82235632/4f176516-17c7-42a5-9cab-51fad7184cdc)
+
+> **UPDATE**: On the **2023-11-08** Node and Dashboard limits will be applied on the Netdata Cloud **Community plan**, while all current features of the Community plan will remain the same.
+
+<!--truncate-->
+
+# What you need to know?
+
+* On Netdata Cloud Free Community plan, the number of **active nodes** that can be concurrently visualized on the Netdata dashboards, as well as the number of **active custom dashboards** for accounts created after 2023-11-06/07 will be subject to limits.
```
Comment (sashwathn) [inline blog/2023-11-02-netdata-plan-changes/index.md:19]: Let us set a date as 2023-11-06 and it wouldn't matter if it comes a day later?

File: blog/2023-11-02-netdata-plan-changes/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+> **UPDATE**: On the **2023-11-08** Node and Dashboard limits will be applied on the Netdata Cloud **Community plan**, while all current features of the Community plan will remain the same.
+
+<!--truncate-->
+
+# What you need to know?
+
+* On Netdata Cloud Free Community plan, the number of **active nodes** that can be concurrently visualized on the Netdata dashboards, as well as the number of **active custom dashboards** for accounts created after 2023-11-06/07 will be subject to limits. 
+
+* Netdata Cloud accounts on the current Community or Early Bird plans created before **2023-11-07** will not be affected, and no limits will be set.
```
Comment (sashwathn) [inline blog/2023-11-02-netdata-plan-changes/index.md:21]: I would prefer if we do not explicitly mention this so our existing community users reach out to us and we get an opportunity to talk to them.

File: blog/2023-11-02-netdata-plan-changes/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+* Any new Netdata Cloud user sign-up will be automatically enrolled to a **30-day Free Business Trial**, so all can experience Netdata Cloud without any limits or restrictions - no debit or credit cards will be required.
+
+* The changes above do not apply to our other plans: Netdata Cloud Pro, Business, On-prem subscriptions, and Netdata FOSS Agent versions.
```
Comment (sashwathn) [inline blog/2023-11-02-netdata-plan-changes/index.md:25]: Should we talk about Pro explicitly in this statement as we are talking about moving them to Business?

**Review by sashwathn — APPROVED** [review]

File: blog/2023-11-02-netdata-plan-changes/index.md (hunk 1 above)
Comment (hugovalente-pm) [reply blog/2023-11-02-netdata-plan-changes/index.md:19]: why to make it the beg. of the week?

Comment (hugovalente-pm) [reply blog/2023-11-02-netdata-plan-changes/index.md:19]: oh sorry, I missed that it had 06/07... will put 07

File: blog/2023-11-02-netdata-plan-changes/index.md (hunk 3 above)
Comment (hugovalente-pm) [reply blog/2023-11-02-netdata-plan-changes/index.md:25]: yeah probably makes sense to remove it

**Pull request merged**